lib/facter/pe_status_check.rb

```diff
@@ -213,6 +216,21 @@
     end
   end
 
+  chunk(:S0020) do
+    Are All Services running
+    next unless ['primary', ].include?(Facter.value('pe_status_check_role'))
```

Comment:
the comma beside primary is not needed if its not a list, this will cause it to fail